src/app/middlewares/token.validator.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
+import { config } from '../config/config';
+
+export default function (request, response, next) {
+    let token = request.headers['x-access-token'] || request.headers['authorization'];
+
+    if (token) {
```

Comment:
Rather than using `if/else` condition you can use simply `if` with negative case and after it proceed with positive like
```
...some code...
if (!token) {
  ...describe smth...
  return ;
}
...here proceed with some positive case...
```